easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'AtomPAW'
+version = '4.1.0.5'
```

Comment:
@jmbeuken This easyconfig for `AtomPAW` 4.1.0.5 isn't being used to resolve a dep, and should perhaps be removed since it's basically replaced by the one for `4.1.0.6` version?

Reply:
ok ! 4.1.0.5 removed...